ladybug_display_schema/visualization.py

```diff
@@ -293,28 +286,75 @@ class GraphicContainer(NoExtraBaseModel):
         'If None, the default units of the data_type will be used.'
     )
 
+
+class AnalysisGeometry(NoExtraBaseModel):
+    """An object where multiple data streams correspond to the same geometry."""
+
+    type: constr(regex='^AnalysisGeometry$') = 'AnalysisGeometry'
+
+    geometry: Union[Mesh2D, Mesh3D, Polyface3D, List[GEOMETRY_UNION]] = Field(
+        ...,
+        description='A ladybug-geometry object or list of ladybug-geometry objects that '
+        'is aligned with the values in the input data_sets. If a Mesh or Polyface '
+        'is specified here, it is expected that the number of values match the '
+        'number of faces or the number of vertices. If a list of geometry objects '
+        'is specified (ie. a list of Point3Ds), it is expected that the length '
+        'of this list align with the number of values.'
+    )
+
+    data_sets: List[VisualizationData] = Field(
+        ...,
+        min_items=1,
+        description='An list of VisualizationData objects representing the data sets '
+        'that are associated with the input geometry.'
+    )
+
+    default_data: int = Field(
```

Comment:
I just checked and in VTK this is called active_data. Might be more explicit as to what this number does.

Reply:
I like it. It's much more oriented to the perspective of an object that is actively being used rather than something static.